components/google-cloud/google_cloud_components/aiplatform/__init__.py

```diff
@@ -13,18 +13,74 @@
 # limitations under the License.
 """Core modules for AI Platform Components."""
 
-from google.cloud import aiplatform as aip
+from google.cloud import aiplatform
 from google_cloud_components.aiplatform import utils
 
-DatasetCreateOp = utils.convert_method_to_component(aip.Dataset.create)
+DatasetCreateOp = utils.convert_method_to_component(
+	aiplatform.Dataset.create)
 
 DatasetExportDataOp = utils.convert_method_to_component(
-    aip.Dataset.export_data, should_serialize_init=True)
+	aiplatform.Dataset.export_data,
```

Comment:
could we replace tab with space, also similar comment on trailing white space, and blank line with white space.

```
py38 run-test: commands[1] | flake8 . --count --select=E9,F63,F7,F82,W --show-source --statistics
./google_cloud_components/aiplatform/remote_runner.py:42:1: W293 blank line contains whitespace
    
^
./google_cloud_components/aiplatform/remote_runner.py:48:1: W293 blank line contains whitespace
    
...
./google_cloud_components/aiplatform/utils.py:391:81: W291 trailing whitespace
            # perhaps, another option is to embed in yaml as json seralized list 
                                                                                ^
27    W191 indentation contains tabs
7     W291 trailing whitespace
1     W292 no newline at end of file
11    W293 blank line contains whitespace
```